app/build.gradle

```diff
@@ -48,6 +49,12 @@ dependencies {
     // 参考 https://qiita.com/ymmtyuhei/items/2cb79031e20e1dcff6fc
     testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.0.0-RC1'
     testImplementation 'org.mockito:mockito-android:2.22.0'
+
+    androidTestImplementation 'androidx.test:rules:1.1.0'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/aiphone-test-muramatsu/ChatApp/blob/d937595d69e9ad871c2b5ab863c45c1ee8ea7816/app/build.gradle#L53"></span>A newer version of androidx.test:rules than 1.1.0 is available: 1.2.0</del></td>
    </tr>
  </tbody>
</table>